src/workflow_docs/fetch_docs_sources.sh

```diff
@@ -16,18 +16,29 @@ git clone --no-checkout --depth=1 --single-branch --branch=master --filter=tree:
 git clone --no-checkout --depth=1 --single-branch --branch=main   --filter=tree:0 https://github.com/microbiomedata/metaT /tmp/clones/metaT                         && cd /tmp/clones/metaT             && git sparse-checkout set --no-cone /docs && git checkout
+git clone --no-checkout --depth=1 --single-branch --branch=main   --filter=tree:0 https://github.com/microbiomedata/metaT_ReadsQC /tmp/clones/metaT_ReadsQC         && cd /tmp/clones/metaT_ReadsQC     && git sparse-checkout set --no-cone /docs && git checkout
+git clone --no-checkout --depth=1 --single-branch --branch=main   --filter=tree:0 https://github.com/microbiomedata/metaT_Assembly /tmp/clones/metaT_Assembly       && cd /tmp/clones/metaT_Assembly    && git sparse-checkout set --no-cone /docs && git checkout
+git clone --no-checkout --depth=1 --single-branch --branch=master --filter=tree:0 https://github.com/microbiomedata/metaT_ReadCounts /tmp/clones/metaT_ReadCounts   && cd /tmp/clones/metaT_ReadCounts  && git sparse-checkout set --no-cone /docs && git checkout
 git clone --no-checkout --depth=1 --single-branch --branch=master --filter=tree:0 https://github.com/microbiomedata/metaPro /tmp/clones/metaPro                     && cd /tmp/clones/metaPro           && git sparse-checkout set --no-cone /docs && git checkout
 git clone --no-checkout --depth=1 --single-branch --branch=master --filter=tree:0 https://github.com/microbiomedata/metaMS /tmp/clones/metaMS                       && cd /tmp/clones/metaMS            && git sparse-checkout set --no-cone /docs && git checkout
 git clone --no-checkout --depth=1 --single-branch --branch=master --filter=tree:0 https://github.com/microbiomedata/enviroMS /tmp/clones/enviroMS                   && cd /tmp/clones/enviroMS          && git sparse-checkout set --no-cone /docs && git checkout
 
 # Arrange the "docs" directories into chapters.
 mkdir -p /tmp/book/src/chapters
-mv /tmp/clones/ReadsQC/docs           /tmp/book/src/chapters/1_RQC
-mv /tmp/clones/ReadbasedAnalysis/docs /tmp/book/src/chapters/2_ReadAnalysis
-mv /tmp/clones/metaAssembly/docs      /tmp/book/src/chapters/3_MetaGAssembly
-mv /tmp/clones/mg_annotation/docs     /tmp/book/src/chapters/4_MetaGAnnotation
-mv /tmp/clones/metaMAGs/docs          /tmp/book/src/chapters/5_MAG
-mv /tmp/clones/metaT/docs             /tmp/book/src/chapters/6_MetaT
-mv /tmp/clones/metaPro/docs           /tmp/book/src/chapters/7_Metaproteomics
-mv /tmp/clones/metaMS/docs            /tmp/book/src/chapters/8_Metabolomics
-mv /tmp/clones/enviroMS/docs          /tmp/book/src/chapters/9_NOM
+#
+# TODO: Include Chapter 1, the end-to-end MetaG chapter.
+#       Its source files will reside in this `docs` Git repository.
+#       Its chapter slug will be `1_Metagenome_Workflow_Overview`.
```

Comment:
@eecavanna that has been merged into this repo as content/nmdc/src/reference/metag_wrapper_doc.rst. Im not sure how to link that appropriately here

Reply:
OK, thanks. I didn't realize this chapter name was a reference to that.

In that case, I'd like to leave that specific chapter for a separate PR (I can work on it right after this one gets merged in). The reason is that it will involve making changes to the GitHub Actions workflow, which currently does not "checkout" this repository.

Reply:
> In that case, I'd like to leave that specific chapter for a separate PR (I can work on it right after this one gets merged in).

This is a stale preference of mine. I ended up creating [that PR](https://github.com/microbiomedata/docs/pull/71/) based off of this PR's branch, and have merged it into this PR's branch.